app/src/main/java/edu/wpi/first/shuffleboard/app/MainWindowController.java

```diff
@@ -75,6 +76,7 @@
 /**
  * Controller for the main UI window.
  */
+@SuppressWarnings("PMD.GodClass") // TODO refactor this class
```

Comment:
🤦‍♂️

Reply:
I'm honestly surprised this PMD rule got triggered since the diff is so minor

Reply:
https://github.com/search?utf8=%E2%9C%93&q=%40SuppressWarnings%28%22PMD.GodClass%22%29+extension%3Ajava&type=Code

Reply:
Going to do that refactor after the currently open PRs are merged in. Don't want to force merge conflicts if I can avoid it